playwright-e2e/playwright.config.ts

```diff
@@ -74,12 +74,12 @@ const testConfig: PlaywrightTestConfig = {
     // baseURL: 'http://localhost:3000',
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: process.env.CI ? 'on-first-retry' : 'retain-on-failure',
+    trace: 'retain-on-failure',
```

Comment:
I want to see failed test trace and screenshots when it fails on first run. Sometimes tests pass on second run (rerun).  This is a temporary change and I will revert once all flaky things are resolved.